src/gpu/xr/SDL_openxrdyn.c

```diff
@@ -0,0 +1,178 @@
+/*
+  Simple DirectMedia Layer
+  Copyright (C) 1997-2024 Sam Lantinga <slouken@libsdl.org>
+
+  This software is provided 'as-is', without any express or implied
+  warranty.  In no event will the authors be held liable for any damages
+  arising from the use of this software.
+
+  Permission is granted to anyone to use this software for any purpose,
+  including commercial applications, and to alter it and redistribute it
+  freely, subject to the following restrictions:
+
+  1. The origin of this software must not be misrepresented; you must not
+     claim that you wrote the original software. If you use this software
+     in a product, an acknowledgment in the product documentation would be
+     appreciated but is not required.
+  2. Altered source versions must be plainly marked as such, and must not be
+     misrepresented as being the original software.
+  3. This notice may not be removed or altered from any source distribution.
+*/
+
+#include "SDL_internal.h"
+
+#include "SDL_openxrdyn.h"
+
+#ifdef HAVE_GPU_OPENXR
+
+#if defined(SDL_PLATFORM_APPLE)
+#define SDL_GPU_OPENXR_DYNAMIC "libopenxr_loader.dylib"
+#elif defined(SDL_PLATFORM_WINDOWS)
+#define SDL_GPU_OPENXR_DYNAMIC "openxr_loader.dll"
+#else
+#define SDL_GPU_OPENXR_DYNAMIC "libopenxr_loader.so.1"
```

Comment:
I think you have to do this for Android, and also for all non-Linux unix platforms.

```suggestion
#define SDL_GPU_OPENXR_DYNAMIC "libopenxr_loader.so.1,libopenxr_loader.so"
```
(this suggestion is probably incompatible with the code below because it is a comma-separated list itself)

Reply:
Applied this along with a patch to make it always tokenize the string, so this works

Reply:
I've asked about this at https://github.com/KhronosGroup/OpenXR-SDK-Source/issues/529 btw